docs/features/datastores/1_Capabilities/1_README.md

```diff
@@ -1 +1,9 @@
-FILL ME
+---
+title: Overview
+sidebar_label: Overview
+---
+
+# Current Datastore Capabilities
+
+**QUESTION: What exactly is supposed to go here?? I think i filled in most of the text
```

Comment:
My thought is that there could be a more general introduction to what these features are. For instance once we get diffing the explanation for how it works can be here so it is front and center. 

As an example I don't recall seeing the following explained in this PR.
1. What is private-s3. Like how does it work, benefits of using it, etc.

The current capabilities as noted in this PR (really good btw thank you for this) reads more like examples of how to use said features. I'd just like to give an upfront expectation about the caveats, intended use, and benefit of our capabilities. We don't want to speak to the technical implementation details so it will be a balancing act of giving enough clarity where the utility of the feature is clear, evident, and actionable while lot giving away our secret sauce. If that makes sense?

@essiequoi thoughts, if you are in disagreement we can remove etc.

Reply:
@rlizzo ^

Reply:
we can add this later